src/osp/CoordinateSpaces/CartesianSimple.cpp

```diff
@@ -0,0 +1,82 @@
+/**
+ * Open Space Program
+ * Copyright © 2019-2020 Open Space Program Project
```

Comment:
2021

Reply:
We're over half a year overdue for this one. Should these have been changed as each file was touched? Or how about in a separate PR?

Reply:
There are a few different strategies.

One is to do a once-yearly PR that updates all the timestamps.

The other is to update the files as the files get updated.

I guess just doing a once-yearly PR makes the most sense to me.

Reply:
Not sure if this is important, but if a file remains unchanged when the year is updated, the file becomes different even though it's functionally the same. This makes me prefer the latter option.

On the meantime, I'll add 2021 to only the newly added files

Reply:
The reason for doing the yearly bump would just to have it done as a separate identifiable commit.

There's not a need to only modify the year when the files have changed, as the intention of modifying the date is to indicate that the over-all work is still being maintained, and not just the one file.